src/components/search-columns.js

```diff
@@ -17,6 +19,46 @@ function renderCheckbox(column, query, onCheckChange) {
   ) : (
     ""
   );
+}*/
+  
+function renderCheckbox(column, query, onCheckChange) {
+  return column && column.property && column.checkbox ? (
+    <Select
+      style={{ width: '100%' }}
+      name={column.property}
+      placeholder={column.filterPlaceholder || ""}
+      defaultValue={query[column.property] || "all"}
+      onChange={(value) => onCheckChange(column.property, value)}
+    >
+      <Option value={'all'}>ALL</Option>
+      <Option value={true}>True</Option>
+      <Option value={false}>False</Option>
+      <Option value={null}>NULL</Option>
```

Comment:
Lets change the naming of these to:

All
True
False
Undefined

To be more clear and follow better casings.

Reply:
@tvarsis I will remove the commented code but for this part.